src/main/java/mat/client/shared/MatContext.java

```diff
@@ -233,9 +233,9 @@
 
     private Map<String, String> expressionToReturnTypeMap = new HashMap<>();
 
-    private FeatureFlag matOnFHIR;
+	private List<FeatureFlag> featureFlagList;
```

Comment:
I would suggest to store a MAP here, instead of a list

Reply:
Changed it to MAP